modelplugin/yang/Stratum-1.0.0/ietf-interfaces@2014-05-08.yang

```diff
@@ -0,0 +1,118 @@
+/*
+ * This file has been modified to remove the 2 top level containers
+ * This file is needed only for its type definitions, used in the
+ * openconfig models. If you are using generator - rename the file
```

Comment:
I think the file should end in .modified rather than .yang, as per the comment ... or change the comment

Reply:
done